desktop/src/main/resources/bisq.css

```diff
@@ -109,7 +109,23 @@
 }
 
 .super-large-text {
-    -fx-font-size: 4.1em;
+    -fx-font-size: 4.1em !important;
```

Comment:
Why is `!important` needed here?

Reply:
[`headlineLabel`](https://github.com/bisq-network/bisq2/blob/24e8e0a8b75060b168abc660017303668e4b8dbd/desktop/src/main/java/bisq/desktop/common/view/TabView.java#L51) has [`.bisq-content-headline-label`](https://github.com/bisq-network/bisq2/blob/24e8e0a8b75060b168abc660017303668e4b8dbd/desktop/src/main/resources/bisq.css#L307-L311) class with `-fx-font-size: 1.78em;`. On getting started screen `headlineLabel` should be bigger. I decided to manage it by toggling `.super-large-text`  utility class. Unfortunately without `!important` javafx uses font size from `.bisq-content-headline-label`. 

Using `!important` for utility classes is rather [good practice](https://css-tricks.com/when-using-important-is-the-right-choice/).